src/promptflow/promptflow/executor/_line_execution_process_pool.py

```diff
@@ -196,6 +283,20 @@ def __enter__(self):
 
         self._inputs_queue = Queue()
         self._n_process = self._determine_worker_count()
+        if self._use_fork:
+            self._input_queues = [manager.Queue() for i in range(self._n_process)]
+            self._output_queues = [manager.Queue() for i in range(self._n_process)]
+            self._restart_queue = manager.Queue()
```

Comment:
Need to change it to sth like "_control_signal_queue", since it does both deletion and restarting.

Reply:
fixed